<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniModeWidget</class>
 <widget class="QWidget" name="MiniModeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5,5,5,5,5,5,5,5,5,5,5">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>1</number>
     </property>
     <item>
      <widget class="AeroButton" name="mode_lsb">
       <property name="text">
        <string>LSB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_usb">
       <property name="text">
        <string>USB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_dsb">
       <property name="text">
        <string>DSB</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_cwl">
       <property name="text">
        <string>CWL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_cwu">
       <property name="text">
        <string>CWH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_fm">
       <property name="text">
        <string>FM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_am">
       <property name="text">
        <string>AM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_digu">
       <property name="text">
        <string>DIGU</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_digl">
       <property name="text">
        <string>DIGL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_spec">
       <property name="text">
        <string>SPEC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_sam">
       <property name="text">
        <string>SAM</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="AeroButton" name="mode_freedv">
       <property name="text">
        <string>FDV</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AeroButton</class>
   <extends>QPushButton</extends>
   <header location="global">Util/cusdr_buttons.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
